client/src/constants/action_types.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
I'm going to look into this separately. Its a pretty huge library.
https://github.com/acmcarther/catalyst/issues/15